ui/app/appointments/controllers/manage/calendar/appointmentsDayCalendarController.js

```diff
@@ -54,14 +54,21 @@ angular.module('bahmni.appointments')
                     uiCalendarConfig.calendars[calendar].fullCalendar('changeView', view);
                 };
 
-                var isUserPrivilegedToCreateAppointment = function () {
+                function isOwnAppointmentUserPrivilegedToCreateAppointment (privilege, provider) {
```

Comment:
@sai789, Please use same notation like
```suggestion
                var isOwnAppointmentUserPrivilegedToCreateAppointment = function (privilege, provider) {
```